File: DAW1/programacion/cadenas/17.py
def separar_y_juntar(cadena):

    consonantes = ""
    vocales = "aeiouAEIOU"
    sonvocales = ""

    for i in cadena:
        if i != " ":
            if i in vocales:
                sonvocales += i
            else:
                consonantes += i
    
    return consonantes + sonvocales

print(separar_y_juntar("Hola soy Alvaro Cruces Martinez, me gusta entrenar calistenia y jugar videojuegos"))

File: DAW1/lenguaje/JavaScript/boletin_iniciacion/boletin1-20/17.js
function contar(cadena) {
    let contador = 0;
    for (let i = 0; i < cadena.length; i++) {
        if(cadena[i] !== " ") { 
            contador++;
        }
    }
    return contador;
}

console.log(contar("Hola Mundo"));